src/daft-sql/src/modules/map.rs

```diff
@@ -1,11 +1,33 @@
 use super::SQLModule;
-use crate::functions::SQLFunctions;
+use crate::{
+    functions::{SQLFunction, SQLFunctions},
+    invalid_operation_err,
+};
 
 pub struct SQLModuleMap;
 
 impl SQLModule for SQLModuleMap {
-    fn register(_parent: &mut SQLFunctions) {
-        // use FunctionExpr::Map as f;
-        // TODO
+    fn register(parent: &mut SQLFunctions) {
+        parent.add_fn("map_get", MapGet);
+        parent.add_fn("map_extract", MapGet);
```

Comment:
Is this an alias to maintain ANSI/postgres compatibility?

Reply:
AFAIK, ANSI doesn't really have any standard for what you can/cant call your functions. I mostly added the alias because I've seen it in other db systems (such as duckdb).